gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -230,15 +230,16 @@ public void orchestrate(Spec spec, Properties jobProps, long triggerTimestampMil
       Map<String, String> flowMetadata = TimingEventUtils.getFlowMetadata(flowSpec);
       String flowExecutionId = String.valueOf(FlowUtils.getOrCreateFlowExecutionId(flowSpec));
 
+      // Note we use "" as jobName for lease arbitration since orchestration is at a flow level
```

Comment:
suggest named constant for use of `""` for this particular (`jobName`) purpose

Reply:
how about using `JobStatusRetriever.NA_KEY` which is used for similar purpose